<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="windowModality">
   <enum>Qt::WindowModality::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1612</width>
    <height>637</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>10</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>35</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LayoutDirection::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0,3,0">
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>10</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1866</width>
       <height>266</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #759FBC;
border-radius: 50px; /* Adjust the radius to create an oval shape */
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SizeConstraint::SetMinAndMaxSize</enum>
      </property>
      <property name="topMargin">
       <number>11</number>
      </property>
      <property name="rightMargin">
       <number>27</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_6" native="true">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>34</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">	color: rgba(0, 0, 0, 0.8);
</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;
&lt;head/&gt;
&lt;body&gt;
    &lt;div&gt;
        &lt;span style=&quot;color: rgba(0, 0, 0, 0.8)&quot;&gt;Welcome to&lt;/span&gt;
    &lt;/div&gt;
    &lt;div style=&quot;color: white;&quot;&gt;
        Noteworthy
    &lt;/div&gt;
&lt;/body&gt;
&lt;/html&gt;
</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="name">
        <property name="font">
         <font>
          <pointsize>80</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">	color: rgba(0, 0, 0, 0.8);
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1866</width>
       <height>651</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:  #F0F9FF;
;
border-radius: 50px; /* Adjust the radius to create an oval shape */
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="leftMargin">
       <number>60</number>
      </property>
      <property name="topMargin">
       <number>60</number>
      </property>
      <property name="rightMargin">
       <number>60</number>
      </property>
      <property name="bottomMargin">
       <number>60</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,1">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #757397;
border-radius: 60px; </string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="leftMargin">
              <number>20</number>
             </property>
             <property name="rightMargin">
              <number>20</number>
             </property>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="font">
                <font>
                 <pointsize>35</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">	color: rgba(0, 0, 0, 0.7);
</string>
               </property>
               <property name="text">
                <string>   Join Room</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_5">
               <property name="styleSheet">
                <string notr="true">
    border-radius: 50px;    /* Make the area circular */
    background-color: white; /* Optional: Set background color */
 width: 127px;          /* Set the width of the circular area */
    height: 136px;         /* Set the height of the circular area */
    color: #757397;         /* Set text color to purple */
    text-align: center;    /* Center the text */
  font-size: 120px;      /* Set the font size */</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_6">
               <property name="styleSheet">
                <string notr="true">
    border-radius: 50px;    /* Make the area circular */
    background-color: white; /* Optional: Set background color */
 width: 127px;          /* Set the width of the circular area */
    height: 136px;         /* Set the height of the circular area */
    color: #757397;         /* Set text color to purple */
    text-align: center;    /* Center the text */
  font-size: 120px;      /* Set the font size */</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_7">
               <property name="styleSheet">
                <string notr="true">
    border-radius: 50px;    /* Make the area circular */
    background-color: white; /* Optional: Set background color */
 width: 127px;          /* Set the width of the circular area */
    height: 136px;         /* Set the height of the circular area */
    color: #757397;         /* Set text color to purple */
    text-align: center;    /* Center the text */
  font-size: 120px;      /* Set the font size */</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_8">
               <property name="styleSheet">
                <string notr="true">
    border-radius: 50px;    /* Make the area circular */
    background-color: white; /* Optional: Set background color */
 width: 127px;          /* Set the width of the circular area */
    height: 136px;         /* Set the height of the circular area */
    color: #757397;         /* Set text color to purple */
    text-align: center;    /* Center the text */
  font-size: 120px;      /* Set the font size */</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_9">
               <property name="styleSheet">
                <string notr="true">
    border-radius: 50px;    /* Make the area circular */
    background-color: white; /* Optional: Set background color */
 width: 127px;          /* Set the width of the circular area */
    height: 136px;         /* Set the height of the circular area */
    color: #757397;         /* Set text color to purple */
    text-align: center;    /* Center the text */
  font-size: 120px;      /* Set the font size */</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="codeEnter">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>110</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: white;

</string>
               </property>
               <property name="text">
                <string>&gt;</string>
               </property>
               <property name="iconSize">
                <size>
                 <width>100</width>
                 <height>100</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_4" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: #90C3C8;
border-radius: 50px;
font-size: 14px;

</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="leftMargin">
              <number>32</number>
             </property>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>400</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>35</pointsize>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgba(0, 0, 0, 0.7);
font: 35pt;
  border-radius:  50px;
</string>
               </property>
               <property name="text">
                <string>Create room</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="newpage">
               <property name="styleSheet">
                <string notr="true">border-radius: 26px;
</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset theme="QIcon::ThemeIcon::ListAdd"/>
               </property>
               <property name="iconSize">
                <size>
                 <width>100</width>
                 <height>100</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_5" native="true">
            <property name="styleSheet">
             <string notr="true"> background-color: #C5E3F7;
  border-radius:  50px;
</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QPushButton" name="changename">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>35</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
    background-color: #C5E3F7;
    border-radius:  60px;
	color: rgba(0, 0, 0, 0.7);
}
</string>
               </property>
               <property name="text">
                <string>Change name</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_7" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>35</pointsize>
          <fontweight>Light</fontweight>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgba(144, 195, 200, 0.4);
</string>
        </property>
        <property name="text">
         <string>v0.0.1(1)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="error">
        <property name="font">
         <font>
          <pointsize>25</pointsize>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgba(0, 0, 0, 0.7);
font: 25pt;
border-radius:  50px;
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
